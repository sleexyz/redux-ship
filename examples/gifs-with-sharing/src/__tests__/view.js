// @flow
import React from 'react';
import * as Test from '../test';
import * as Model from '../model';
import Index from '../view';

const defaultProps = {
  dispatch: Test.dispatch,
  state: Model.initialState,
};

Test.snapshotComponent(Index, {
  'default': defaultProps,
});
