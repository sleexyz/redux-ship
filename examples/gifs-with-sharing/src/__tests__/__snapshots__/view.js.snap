exports[`test default 1`] = `
<div
  className="Index">
  <div
    className="Index-header">
    <img
      alt="logo"
      className="Index-logo"
      src="test-file-stub" />
    <h2>
      Redux Ship
    </h2>
  </div>
  <p>
    Open the console to see the logs. Sources are 
    <a
      href="https://github.com/clarus/redux-ship/tree/master/examples/gifs">
      on GitHub
    </a>
    .
  </p>
  <h1>
    Counter
  </h1>
  <div>
    <p>
      Count: 
      0
    </p>
  </div>
  <h1>
    Gifs
  </h1>
  <div
    className="Index-randomGif">
    <div
      className="RandomGif">
      <img
        alt="no gifs"
        className="RandomGif-picture"
        src="test-file-stub" />
      <button
        disabled={false}
        onClick={[Function anonymous]}>
        New cat
      </button>
    </div>
    <div
      className="RandomGif">
      <img
        alt="no gifs"
        className="RandomGif-picture"
        src="test-file-stub" />
      <button
        disabled={false}
        onClick={[Function anonymous]}>
        New minion
      </button>
    </div>
    <div
      className="RandomGif">
      <img
        alt="no gifs"
        className="RandomGif-picture"
        src="test-file-stub" />
      <button
        disabled={false}
        onClick={[Function anonymous]}>
        New dog
      </button>
    </div>
    <div
      className="RandomGif">
      <img
        alt="no gifs"
        className="RandomGif-picture"
        src="test-file-stub" />
      <button
        disabled={false}
        onClick={[Function anonymous]}>
        New gify
      </button>
    </div>
  </div>
</div>
`;
